src/esp/physics/bullet/BulletRigidObject.cpp

```diff
@@ -373,6 +373,20 @@ const Magnum::Vector3 BulletRigidObject::getInertia() {
   }
 }
 
+const Magnum::Matrix3 BulletRigidObject::getInertiaMatrix() {
+  if (rigidObjectType_ == SCENE) {
+    const Magnum::Matrix3 inertia = Magnum::Matrix3();
+    return inertia;
+  } else {
+    const Magnum::Vector3 vecInertia = getInertiaVector();
+    const Magnum::Matrix3 inertia =
+        Magnum::Matrix3(Magnum::Vector3(vecInertia.x(), 0, 0),
+                        Magnum::Vector3(0, vecInertia.y(), 0),
+                        Magnum::Vector3(0, 0, vecInertia.z()));
```

Comment:
Not complaining, just a suggestion: there's [Matrix3::fromDiagonal()](https://doc.magnum.graphics/magnum/classMagnum_1_1Math_1_1RectangularMatrix.html#a2b5423dcaab476595937428092187123) so you don't need to type it all out ;)